### ISSUE TITLE
Kolbot Crush everytime after Chestmania

### ISSUE THREAD

**Issue by Kaktoo** [kolton/d2bot-with-kolbot#1140]
Hello guys I am new here but need a little bit help. Sorry for my Englisch

I am using Kolbot with no problem but at one script they make some problems. When i Turn Chestmania on True and they do the last chest in act 5, go town and sold some random items. They dont exit game, 
instead they close diablo window and show me these ' 

Window has unexpectedly exited... starting profile
Crash Info: Script: no entry Area: Harrogath

How can I fix it ? I try every think, New version Kolbot and other Chars but its end same. 

Please Help

**Comment by fabio1999ita** [comment]
do you have downloaded last version of kolbot and set d2bot.exe and game.exe whit administrative privileges and compatibility whit windows 7?

**Comment by Kaktoo** [comment]
Yes all done but no change

**Comment by Kaktoo** [comment]
Yes I have downloaded it at Blizzard with no cd rom. At start everythink was fine but since 2weeks everytime after chestmania they crash. Everygame, no matter which char or Kolbot version

**Comment by fabio1999ita** [comment]
I don't know, try to add a fast and easy script after chest, i don't know like pindleskin or how it write

**Comment by Kaktoo** [comment]
Do not works...

**Comment by iihieiii** [comment]
### Hi, i have made some changes and they seems to have fixed the problem: 


:arrow_right: **1)** FIRST please Check if ***D2bot.exe*** and ***Game.exe*** TRULY are in ***"Executable as Admin" and "Compatiblity mode with WIN XP SERVICE PACK 2"*** (mine was on 3), and also apply this for all user on : Properties/Security tab/Edit, and then check if all user can do this ;)

### I know all this things seems **_superstitious_** BUT after all this changes im runnig the bot for 2 hours and had only 1 crash lol, way better than the crash every minute lol.


***---- NOW THE REAL CHANGES ------***

:arrow_right: **2)** Check if the folders on: 
//d2bot-with-kolbot-master\d2bs\kolbot\libs\ ***config***
//d2bot-with-kolbot-master\d2bs\kolbot\ ***pickit***

Have no other Files or Folders beyond the originals ( mine had alot folders that i made like little ''backups'' files for edeting kolton.nip), if you have alot trash files like me, just place them to another location in you desktop.

:arrow_right: **3)** In the folder : ***d2bot-with-kolbot-master\d2bs\kolbot\data***, if you have changed the files to restart runs counting ( like i did sometimes lol), i recommend you to make a new Profile and delete the older one ;)

:arrow_right: **4)** Set ***Config.ItemInfoQuality = [ ];***, on your "char name".js file, has no numbers at all (like this [ ]; ) because i had put some numbers there but i forgot to remove them :(

:arrow_right: **5)** Set ***Config.LogExperience = false;*** , for some reason i sense that this makes more crashes when is set to True.

:arrow_right: **6)** If you are runing the ***Scripts.Cows = true;*** , try to put this script has the last script to run, just after the last Baal script.

:arrow_right: **7)** If you use this line in your kolton.nip for keys : ***( [name] == key # # [maxquantity] == 12)*** , try to put this line in the end of your kolton.nip file, has the last line of command.

:arrow_right: **8)** And the last but not least, ***DON`T RUN the d2bot.exe FROM A SHORTCUT*** (even if its from your desktop! )
Open your folder **d2bot-with-kolbot-master** and double-click d2bot.exe from there! ;)


***I know all this things seems **_superstitious_** BUT after all this changes im runnig the bot for 2 hours and had only 1 crash lol, way better than the crash every minute lol.***

![Image of kolbot no crashs](https://i.imgur.com/2X2wFId.gif)

:heavy_check_mark: ***Please try it out ;)*** :heavy_check_mark:
